src/Infrastructure.VS/Infrastructure.VS.csproj

```diff
@@ -8,10 +8,13 @@
     <RequiresSigning>true</RequiresSigning>
   </PropertyGroup>
 
-  <ItemGroup Label="Framework references">
-  </ItemGroup>
-
   <ItemGroup Label="Package references">
+    <Reference Include="System.ComponentModel.Composition" />
+    <PackageReference Include="Microsoft.VisualStudio.Shell.14.0" Version="14.3.25407" />
+    <PackageReference Include="Microsoft.VisualStudio.Shell.Immutable.10.0" Version="10.0.30319" />
```

Comment:
`Immutable.10` isn't referenced by `Integration.Vsix` (although it does appear in the `bin` folder, so perhaps it was pulled in transitively). 
Does it definitely required here?

Reply:
(doh! - commenting on a draft. I'll stop until you request a review).

Reply:
FYI you were right, it wasn't needed.